Source/LoggingEvent.swift

```diff
@@ -0,0 +1,22 @@
+public class LoggingEvent {
```

Comment:
Any reason this is a class? Also, any reason that all of its properties are mutable?